ui/src/components/grabbable/QasGrabbable.vue

```diff
@@ -61,17 +66,20 @@ function handleEnableScrollOnGrab () {
 function initScrollOnGrab () {
   if (hasScrollOnGrab.value) return
 
-  scrollOnGrab.value = setScrollOnGrab(grabContainer.value, {
-    onGrabFn: onGrab,
-    onMoveFn: setGrabPosition,
-    onScrollFn: setGrabPosition
-  })
+  scrollOnGrab.value = setScrollOnGrab(
```

Comment:
Por que a mudança?

Reply:
só to passando a nova propriedade do helper, por ser 3 parametros agr, melhorei a indentação apenas